### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by shafran-andriy**

### CODE REVIEW

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Rewrite your function to one line with using method `count` of string

**Comment by shafran-andriy** [comment]
Function has been rewrited with using method count of string

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Fix flake8, please

File: app/main.py [hunk 1]
```diff
@@ -10,8 +10,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     count_occurrences("abc", "d") == 0
     count_occurrences("ABC", "a") == 1
 
+    
+
```
Comment (EdAlekseiev) [inline app/main.py:14]: Delete these lines
```suggestion
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Flake8 still has not been fixed

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Fix flake8

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**